### PR TITLE
Restrict mimetypes of files in admin

### PR DESCRIPTION
**Pull request by eodolphi**
If applicable

- [ ] Added translatable strings to `server-deployment`
- [ ] Added translations to `sever-deployment`

### CODE REVIEW

File: bluebottle/files/models.py [hunk 1]
```diff
@@ -9,12 +9,27 @@
...
 
 class File(AnonymizationMixin, models.Model):
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
     created = models.DateField(_('created'), default=timezone.now)
-    file = models.FileField(_('file'), upload_to='files')
+    file = models.FileField(
+        _('file'),
+        upload_to='files',
+        validators=[FileMimetypeValidator(
+            allowed_mimetypes=[
```
Comment (gannetson) [inline bluebottle/files/models.py:22]: Can we user `settings.PRIVATE_FILE_ALLOWED_MIME_TYPES` here?

Comment (eodolphi) [reply bluebottle/files/models.py:22]: Maybe. Is that list always the same? This includes zips, which might not be a good idea for private files.

Comment (eodolphi) [reply bluebottle/files/models.py:22]: Oh no, this is another thing. Yeah, that sound like a good idea.

File: bluebottle/utils/validators.py [hunk 2]
```diff
@@ -75,3 +79,56 @@ def deconstruct(self):
...
+        if code is not None:
+            self.code = code
+
+    def __call__(self, value):
+        mimetype = mime.from_buffer(value.file.read(1000))
+        value.file.seek(0)
+
+        if self.allowed_mimetypes is not None and mimetype not in self.allowed_mimetypes:
+            import ipdb
+            ipdb.set_trace()
```
Comment (gannetson) [inline bluebottle/utils/validators.py:106]: Remove this

**Pull request merged**